ros2launch/ros2launch/command/launch.py

```diff
@@ -79,6 +79,9 @@ def add_arguments(self, parser, cli_name):
         parser.add_argument(
             '-d', '--debug', default=False, action='store_true',
             help='Put the launch system in debug mode, provides more verbose output.')
+        parser.add_argument(
+            '-f', '--log-file-name', default=None,
```

Comment:
i think we can add the default here, so that user can see the default with help option.

```suggestion
            '-f', '--log-file-name', type=str, default='launch',
```